code/online_challenges/src/codechef/STRWN/STRWN.cpp

```diff
@@ -1,59 +1,37 @@
 #include <iostream>
```

Comment:
Looks good! Could you also reformat the file with clang format and 4 spaces? This website can do it: https://zed0.co.uk/clang-format-configurator/